package.json

```diff
@@ -3,8 +3,15 @@
   "version": "0.0.1",
   "description": "this will be sooooo cool",
   "main": "index.js",
-  "scripts": {},
+  "scripts": {
+    "lint": "eslint ./lib/arithmetic.js ./lib/greet.js",
+    "test": "jest"
```

Comment:
Make sure to add coverage and verbose to your test.

"scripts": {
    "lint": "eslint **/*.js",
    "test": "jest --verbose --coverage",
    "test-watch": "jest --watchAll --verbose --coverage"
  },